services/movie.py

```diff
@@ -28,13 +32,14 @@
     genres_ids: list = None,
     actors_ids: list = None,
```

Comment:
Avoid using mutable default arguments like `list`. Use `None` as the default value and initialize the list inside the function if necessary to prevent potential bugs.